templates/module/Source/module/Private/Generated/OLink/olinkclient.cpp.tpl

```diff
@@ -22,19 +22,41 @@
 #include "Engine/Engine.h"
 #include "ApiGear/Public/ApiGearConnectionManager.h"
 #include "Misc/DateTime.h"
+THIRD_PARTY_INCLUDES_START
+#include "olink/clientnode.h"
+#include "olink/iobjectsink.h"
+THIRD_PARTY_INCLUDES_END
 
 using namespace ApiGear::ObjectLink;
 
+class {{ $Iface }}OLinkSinkPrivate : public ApiGear::ObjectLink::IObjectSink
+{
+public:
+	{{ $Iface }}OLinkSinkPrivate({{$Class}}& parent);
+
+	// olink sink interface
+	std::string olinkObjectName() override;
```

Comment:
To make this class not generated this would have to be passed in ctor, and stored as a member